lib/Ama.pm

```diff
@@ -96,6 +102,7 @@ $self->plugin("OAuth2Accounts" => {
   $r->delete('/questions/:question_id')->to('questions#remove')->name('remove_question'); # Delete from DB and redirect to questions
   $r->delete('/removeAll')->to('questions#removeAll')->name('removeAll'); #Delete every question
   $r->get('/questions/:creator/:answered/:orderby/:direction/:limit/:keyword')->to('questions#getQuestions')->name('get_answered');
+  $r->post('/feedback_comment')->to('emailfeedback#submit')->name('submit_feedback');
```

Comment:
I think the API should be available on just /feedback.  In fact, the feature is feedback, and the post parameter might be comment.  Right now the post parameter is also called feedback.  This would be ok, but it does make it difficult to distinguish between the control/action and the data being passed.  Perhaps use feedback for the controller/action and comment for the data being passed.